docs/reference/commandline/inspect.md

```diff
@@ -13,14 +13,13 @@ parent = "smn_cli"
 ```markdown
 Usage:  docker inspect [OPTIONS] NAME|ID [NAME|ID...]
 
-Return low-level information on one or multiple containers, images, volumes,
-networks, nodes, services, or tasks identified by name or ID.
+Return low-level information on docker object(s) (e.g. container, image, volume,
+network, node, service, or task) identified by name or ID
```

Comment:
Can you update the man page as well? https://github.com/docker/docker/blob/master/man/docker-inspect.1.md